tests/kani/AsyncAwait/main.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+// compile-flags: --edition 2018
```

Comment:
Please add a comment saying what you are testing

Reply:
Done